src/app/main/error/error.component.ts

```diff
@@ -8,14 +17,90 @@ import { Title } from '@angular/platform-browser';
 })
 export class ErrorComponent implements OnInit {
 
-    @Input() status?: number = 404;
+    @Input() status: number;
+
+    // default error messages
+    errorMessages: ErrorMsg[] = [
+        {
+            status: 0,
+            message: "Undefined Error",
+            description: `Maybe I'm a teapot, maybe not.<br>
+            But anyway, something undefined went wrong.`,
+            action: 'goback',
+            image: 'dsp-error.svg'
+        },
+        {
+            status: 403,
+            message: "Forbidden",
+            description: `This is not the content you were looking for.<br>
+            Your request was valid but you do not have the<br>
+            necessary permissions to access it.`,
+            action: 'goback',
+            image: 'dsp-error-403.svg'
+        },
+        {
+            status: 404,
+            message: "Not found",
+            description: `This is not the content you were looking for.<br>
```

Comment:
This description can simply say "The content you were looking for cannot be found." No need for the second sentence as it is clear nothing was found.

Reply:
Done in 686f1ce